### DISCUSSION TITLE
Override Metadata source for Anime (Jujutsu Kaisen)

### DISCUSSION THREAD

**Discussion by molinamos** [seerr-team/seerr#910] [Q&A] [answered]
Jujutsu Kaisen (2020) data between the different info providers. All but TMDB show seasons 2. This leads to issues where Sonaarr won't monitor season 2 because it's only been requested season 1.  In the end only half of the episodes are asked for.

Is there a way to change metadata providers locally? Would like a way to fix this for everyone so they don't come accross the same issue.

From TMDB, it doesn't look like they will update it either?
https://www.themoviedb.org/tv/95479/discuss/64a5672ada10f0011cb49f99?page=1

Other sources that match:
- https://ui.eidr.org/view/content?id=10.5240/EC00-1B32-7500-CABE-54D9-V
- https://www.thetvdb.com/series/jujutsu-kaisen#seasons
- https://myanimelist.net/anime/40748/Jujutsu_Kaisen

**Comment by fallenbagel** [comment]
We use tmdb. You cannot override the metadata source.

Tmdb is open source. You can change it yourself. As long as you follow their guidelines

**Reply by molinamos** [reply]
The tvmd forum post mentions that they will not change it. 

For these situations what is the recommendation? I can manually add it to Sonaarr. I would have to hope that Jellyseerr won't remove it later and have to manually add more seasons in the future.

**Reply by fallenbagel** [reply]
> The tvmd forum post mentions that they will not change it. 

Tvmd? I am talking about tmdb. It's open source anyone can change anything by logging in. I have done that myself several times. To fix stuff

Though I would make sure to follow the tmdb bible.

Jellyseerr will always use tmdb. In those situations a manual override with sonarr will work. There is a pr open rn that could allow tvdb support but you'll need your own tvdb api key which you'll have to request from them.

**Reply by fallenbagel** [reply]
> The tvmd forum post mentions that they will not change it. 

Tvmd? I am talking about tmdb. It's open source anyone can change anything by logging in. I have done that myself several times. To fix stuff

Though I would make sure to follow the tmdb bible.

Jellyseerr will always use tmdb. In those situations a manual override with sonarr will work. There is a pr open rn that could allow tvdb support but you'll need your own tvdb api key which you'll have to request from them.

**Reply by molinamos** [reply, marked as answer]
O sorry, getting the acronyms mixed. That forum post was from TMDB, mods saying they won't change it. I don't want to change it in-case they ban me and since that would make the info go against everywhere else. Since this sounds more like a feature request, I'll mark this as answered.

I'll check out that PR. Since Jellyseerr is a fork of Overseer, does it make sense to do the change there and patch/pull it here? From the guideline on contributing it mentions patches but I don't see any in the recent comments.

**Reply by fallenbagel** [reply]
> I'll check out that PR. Since Jellyseerr is a fork of Overseer, does it make sense to do the change there and patch/pull it here? From the guideline on contributing it mentions patches but I don't see any in the recent comments. 

What change?

**Reply by molinamos** [reply]
The PR that allows TVDB support.

**Reply by fallenbagel** [reply]
> The PR that allows TVDB support.

The PR is for jellyseerr?

**Reply by molinamos** [reply]
Yes, I mean for features like that PR, wouldn't something like that benefit both Overseerr and Jellyseerr? Just curious is Jellyseerr pulls patches or features from Overseerr.

**Reply by fallenbagel** [reply]
> Yes, I mean for features like that PR, wouldn't something like that benefit both Overseerr and Jellyseerr? Just curious is Jellyseerr pulls patches or features from Overseerr.

We do pull from overseerr and contribute, but currently, our stack is very different, so a pull is not possible. (We are on nextjs 14, swc instead of babel, pnpm instead of yarn, nodejs 20 instead of 18, and react latest as well) so pulls from overseerr is not possible that easily
